src/pages/HomePage.tsx

```diff
@@ -198,6 +199,22 @@ const cardGroups = [
       },
     ],
   },
+  {
+    group: 'Playground',
+    linkItems: [
+      {
+        to: RouteConst.PLAYGROUND,
+        title: 'Playground',
+        description: 'Drag & Drop playground',
+        startIcon: (
+          <Stack>
+            <DiPlay color="blue" />
+          </Stack>
+        ),
```

Comment:
It's okay, but plz remove Stack component.

startIcon: (
    `<DiPlay color="blue" />`
)